src/renderers/WebGLRenderer.js

```diff
@@ -2012,8 +2013,15 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 		}
 
+		if ( material.envMap instanceof THREE.WebGLRenderTargetCube ) {
+
+			console.warn( 'THREE.WebGLRenderTargetCube: don\'t use render targets as textures. Use their .texture property instead.' );
+			material.envMap = material.envMap.texture;
+
+		}
+
 		uniforms.envMap.value = material.envMap;
-		uniforms.flipEnvMap.value = ( material.envMap instanceof THREE.WebGLRenderTargetCube ) ? 1 : - 1;
+		uniforms.flipEnvMap.value = ( material.envMap && material.envMap.renderTargetCube ) ? 1 : - 1;
```

Comment:
The WebGLRenderTargetCube has a Texture instance because it extends WebGLRenderTarget. CubeTexture is not involved in this check any way. 

To replicate the original logic here, I had to resort to this awkward flag because there is no other way to know whether the texture at hand belongs to a WebGLRenderTargetCube.